hw/ip/otbn/data/otbn_sec_cm_testplan.hjson

```diff
@@ -167,86 +224,126 @@
+            Golden model of OTBN in Python models secure wiping cycle accurately. So in every
+            test at least one internal secure wipe because of exiting a reset. Hence there
+            is no need for a specific test.
+            '''
       stage: V2S
       tests: ["otbn_single"]
     }
     {
       name: sec_cm_write_mem_integrity
-      desc: "Verify the countermeasure(s) WRITE.MEM.INTEGRITY."
+      desc: '''Verify the countermeasure(s) WRITE.MEM.INTEGRITY.
+            DV environment calculates CRC values independently from RTL with every memory write
+            over the bus and than calculates it with the design. otbn_multi does not use backdoor
+            writes to memory so it's guaranteed to see CRC checking for IMEM and DMEM there.
+            '''
       stage: V2S
       tests: ["otbn_multi"]
     }
     {
       name: sec_cm_ctrl_flow_count
-      desc: "Verify the countermeasure(s) CTRL_FLOW.COUNT."
+      desc: '''Verify the countermeasure(s) CTRL_FLOW.COUNT.
+            Golden model of OTBN in Python models instruction counter register cycle accurately.
+            So in every test there is a comparison between model instruction counter value and
+            design instruction counter value. Hence there is no need for a specific test.
+            '''
       stage: V2S
       tests: ["otbn_single"]
     }
     {
       name: sec_cm_ctrl_flow_sca
-      desc: "Verify the countermeasure(s) CTRL_FLOW.SCA."
+      desc: '''Verify the countermeasure(s) CTRL_FLOW.SCA.
+            Since this is related with unused parts of the control path not changing
+            throughout an OTBN run this security countermeasure is verified with assertions.
+            '''
       stage: V2S
       tests: ["otbn_single"]
     }
     {
       name: sec_cm_data_mem_sw_noaccess
-      desc: "Verify the countermeasure(s) DATA.MEM.SW_NOACCESS."
+      desc: '''Verify the countermeasure(s) DATA.MEM.SW_NOACCESS.
+            Read write access using tl_access task is tested with the first Kib of address in
+            DMEM. Expected result is a error response from the TLUL bus.
+            '''
       stage: V2S
       tests: ["otbn_sw_no_acc"]
     }
     {
       name: sec_cm_key_sideload
-      desc: '''Verify the countermeasure(s) SCRAMBLE.KEY.SIDELOAD.
-
-            Simulation can't really prove that the sideload key is unreachable by SW.
-            However, from defined CSRs and memory returned data, there is no way to read
-            scramble key by SW.
+      desc: '''Verify the countermeasure(s) KEY.SIDELOAD.
+            Invalid sideload keys are allowed in the sideload key sequence fifty percent of the
+            time by default. That verifies the sceneario when in the case of an invalid sideload
+            key, OTBN would generate a KEY_INVALID recoverable software error.
+            This happens test agnostic so otbn_single is mapped to represent an OTBN run in general.
```

Comment:
The spec mentions `Keys can be sideloaded without exposing them to the CPU` for this countermeasure.

IMO, DV can't really verify that. @GregAC can also confirm. Perhaps we can mention it here. I'm fine to link the test that uses the sideload key.